src/main/java/frc/robot/commands/ClimbingDown.java

```diff
@@ -5,33 +5,43 @@
 import frc.robot.subsystems.ClimbingArm;
 
 public class ClimbingDown extends Command {
-  ClimbingArm climbArm;
-
-  public ClimbingDown(ClimbingArm climbArm) {
-    this.climbArm = climbArm;
-    addRequirements(climbArm);
-
+  // setActive setActive;
+  ClimbingArm m_climbArm;
+
+  /**
+   * Creates a new ArcadeDrive command.
+   *
+   * @param left       The control input for the left side of the drive
+   * @param right      The control input for the right sight of the drive
+   * @param driveSubsystem The driveSubsystem subsystem to drive
+   */
+  public ClimbingDown(ClimbingArm importedClimbingArm) {
+    m_climbArm = importedClimbingArm;
   }
 
 
   @Override
   public void execute() {
-    climbArm.armDown();
+    m_climbArm.armDown();
 
   }
 
   @Override
-  public void initialize() {
-    if (!climbArm.switchLeft.isOk()||!climbArm.switchRight.isOk()) {
-      CommandScheduler.getInstance().cancel(this);
-
+  public void initialize(){
+    if (!m_climbArm.switchLeft.isOk()||!m_climbArm.switchRight.isOk()){
+        CommandScheduler.getInstance().cancel(this);
```

Comment:
Maybe we add something to to SmartDashboard here like "Elevator switches are borked! OHNO!"?